pyqtgraph/graphicsItems/HistogramLUTItem.py

```diff
@@ -343,15 +444,15 @@ def _showRegions(self):
             self.regions[0].setVisible(True)
             self.gradient.show()
         else:
-            raise ValueError("Unknown level mode %r" %  self.levelMode) 
-    
+            raise ValueError("Unknown level mode {}".format(self.levelMode))
```

Comment:
this is an improvement, but since we're on python 3.7 as a minimum, you could use a f-string if you wanted to

Reply:
Heh, forgot I even did this. f-strings are good for this kind of thing so I'll go ahead and update this and a couple others.